ivy/functional/ivy/general.py

```diff
@@ -2309,8 +2309,83 @@ def gather_nd(
     ret
         New array of given shape, with the values gathered at the indices.
 
+    Examples
+    --------
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([0., 1., 2., 3., 4., 5., 6.])
+    >>> y = ivy.array([1])
+    >>> print(ivy.gather_nd(x, y))
+    ivy.array(1.)
+
+    >>> x = ivy.array([[0., 1., 2.], \
```

Comment:
Based on the logs of the test, the output of this example seems to be incorrect as well, please execute it on your machine and then update the result accordingly.